### DISCUSSION TITLE
Unable to verify signature with FAXXXXXXX script signature

### DISCUSSION THREAD

**Discussion by samleong-wits** [CoolBitX-Technology/coolwallet-js-sdk#453] [Q&A]
Hello CoolBitx, 

I am developing coolwallet SDK to support vechain(VET), i was trying to test my code using signature _"FA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"_ as stated in the README, however i am getting error below.
![image](https://user-images.githubusercontent.com/104709835/177007204-0c4e8167-0f87-4037-9d70-e1baf3e00515.png)

After some investigation, it seems like the test case doesn't execute with the signature above. 
My code is in [vechain branch](https://github.com/sebas-wits/coolwallet-sdk/tree/feature/VeChain). I then execute the ETH coin test case with its signature already (which is the non "FAXXX...") in the source code. I am able to execute the test case, however, once I switched to signature **"FA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"**. I got the same error, see below: 

![image](https://user-images.githubusercontent.com/104709835/177007334-59fe1fe5-558e-46e5-850b-dfed5e4e1447.png)

Why is that?

Thanks

**Comment by hank121314** [comment]
Please ensure that your CoolWallet Pro is already upgraded with `script-tester`.
To check whether your CoolWallet Pro is upgraded, you can try to execute a ETH coin test case.
If the upgrade process succeeds, CoolWallet Pro will display `TEST` and then display `ETH`.
